src/v2/providers/firestore.ts

```diff
@@ -83,6 +101,8 @@ export type DocumentSnapshot = firestore.DocumentSnapshot;
 /** A Firestore QueryDocumentSnapshot */
 export type QueryDocumentSnapshot = firestore.QueryDocumentSnapshot;
 
+type AuthType = "service_account" | "api_key" | "system" | "unauthenticated" | "unknown";
```

Comment:
Can we document when "unknown" will happen and if there are any assumptions the dev can make (e.g. that it's from an admin context)?

Reply:
Also wouldn't hurt to export.

Reply:
Yep the CL is a few months old, quite a few things have changed in the meantime!